src/main/java/com/google/jenkins/plugins/credentials/oauth/JsonServiceAccountConfig.java

```diff
@@ -53,36 +63,109 @@
   private static final long serialVersionUID = 6818111194672325387L;
   private static final Logger LOGGER =
       Logger.getLogger(JsonServiceAccountConfig.class.getSimpleName());
-  private String jsonKeyFile;
+  @CheckForNull
```

Comment:
We'll want to migrate away from using FindBugs as it's no longer maintained. I've created a new issue for this, could you please add a TODO here referencing it: #49

Reply:
Additionally, this class duplicates much of the data/logic contained in this class: https://github.com/googleapis/google-auth-library-java/blob/master/oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java
I've created a new issue tracking de-duplication work with google-auth-library-java project, could you please add a TODO here referencing it: #50

Reply:
You can update the parent pom to get spotbugs for free :smile:

Reply:
https://github.com/jenkinsci/plugin-pom/blob/master/CHANGELOG.md#336

Reply:
@casz Thank you for this, turns out this actually has been building with spotbugs since I already upgraded the parent pom for CasC compatibility.

@craigatgoogle Are you satisfied with that resolution? Added TODO from #50 but not #49 for now.

Reply:
SGTM